source/cursorManager.py

```diff
@@ -59,8 +62,33 @@ def __init__(self, parent, cursorManager, text, caseSensitivity):
 		self.CentreOnScreen()
 		self.findTextField.SetFocus()
 
+	def updateSearchEntries(self, searchEntries, currentSearchTerm):
+		if not currentSearchTerm:
+			return
+		if not searchEntries:
+			searchEntries.insert(0, currentSearchTerm)

```

Comment:
Rather than using 0, please use a named constant. This will help to communicate the ordering of the search entries.
```suggestion
			searchEntries.insert(FIRST_TERM_INDEX, currentSearchTerm)
```

Reply:
The zero here should still be replaced.